File: pi-lens/tests/fixtures/real-bugs/typescript/silent-and-security.ts
export function processUserInput(input: string): string {
	try {
		JSON.parse(input);
	} catch {
		// BUG:silent-error empty catch swallows parser failures
	}

	const api_secret = "hardcoded-secret-value"; // BUG:secrets
	void api_secret;

	// BUG:injection dynamic eval
	return eval(input);
}

File: pi-lens/tests/fixtures/real-bugs/go/silent-and-security.go
package main

import "fmt"

func parseValue(input string) {
	_, err := fmt.Println(input)
	if err != nil {
		// BUG:silent-error swallowed error branch
	}

	api_key := "hardcoded-go-secret" // BUG:secrets
	_ = api_key
}
